lib/experimental/fonts/font-library/class-wp-rest-font-library-controller.php

```diff
@@ -291,6 +291,22 @@ public function uninstall_schema() {
 	public function uninstall_fonts( $request ) {
 		$fonts_param = $request->get_param( 'fontFamilies' );
 
+		if ( empty( $fonts_to_install ) ) {
+			return new WP_Error(
+				'no_fonts_to_install',
+				__( 'No fonts to uninstall', 'gutenberg' ),
+				array( 'status' => 400 )
+			);
+		}
+
+		$needs_write_permission = $this->needs_write_permission( $fonts_to_install );
```

Comment:
I didn't think that we planned to test the directory when uninstalling.
Can we remove this?

If not it looks like fonts_to_(un)install isn't being set.

Reply:
This was addressed in our pairing session. Thanks! Marking this as solved :heavy_check_mark: